src/gc/boehm.cr

```diff
@@ -78,7 +78,7 @@ end
 fun __crystal_malloc64(size : UInt64) : Void*
   {% if flag?(:bits32) %}
     if size > UInt32::MAX
-      raise ArgumentError.new("Negative size")
+      raise ArgumentError.new("size bigger than UInt32::MAX given")
```

Comment:
Also, use an upper case letter to start the exception message?

Reply:
yes please.

Reply:
`size` refers to the argument name

Reply:
oh, then maybe `Given size to malloc is bigger ....`?

Reply:
Fixed